docs/source/overview.rst

```diff
@@ -353,4 +353,17 @@ will handle unit scaling. The correct data will be returned even if
 the tag's position is given in *ms*.
 
 
-TODO: example code
+.. code-block:: python
+
+		x_positions=[2,4,6,8,10,12]
+		tag=block.create_tag('unit example','nix.sampled',x_positions)
+		
+		#single SI unit is supported like mV,cm etc.
+		tag.units="cm"
```

Comment:
actually, this line (362) won't work, at least not in my version :)
The error message, however, is really bad. Thanks for pointing that out..